src/main.ts

```diff
@@ -27,6 +27,7 @@ import 'inter-ui/inter-variable.css';
 import '@unocss/reset/tailwind.css';
 import '@/assets/styles/global.css';
 import 'virtual:uno.css';
+import 'sweetalert2/src/sweetalert2.scss';
```

Comment:
Prueba esto:
```suggestion
import 'sweetalert2/dist/sweetalert2.css';
```

Y si funciona, desinstala ``sass`` con ``npm remove sass`` y así nos quitamos del paso de precompilado CSS.

Reply:
Hecho y funciona.